using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -613,36 +861,46 @@ Here are some examples in Objective-C and Swift:
 **Objective-C**
 
 ```objectivec
-[ACPTarget clearPrefetchCache];
+[ACPTarget resetExperience];
 ```
 
 **Swift**
 
 ```swift
-ACPTarget.clearPrefetchCache()
+ACPTarget.resetExperience()
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### clearPrefetchCache
+
+#### Syntax
+
+```javascript
+resetExperience()
+```
+
+#### Example
 
 **JavaScript**
 
 ```javascript
-ACPTarget.clearPrefetchCache();
+ACPTarget.resetExperience();
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Retrieve Location Content requests
+## retrieveLocationContent
+
+This API to sends a batch request to the configured Target server for multiple mbox locations. 
```

Comment:
Please reword as follows:
"This API sends a batch request to the configured Target server for multiple mbox locations."